test/services_manager_service_test.rb

```diff
@@ -292,66 +292,102 @@
+      context "and is disabled" do
+        before do
+          allow(dbus).to receive(:start_mode).and_return(:manual)
+        end
+
+        it "does not export the service as enabled" do
+          expect(exported_services["enable"]).to_not include("dbus")
+        end
+
+        it "exports the service as disabled" do
+          expect(exported_services["disable"]).to include("dbus")
+        end
       end
     end
 
-    context "when a not static service is disabled" do
+    context "when is not a static sevice (user can enable/disable it)" do
       before do
         allow(dbus).to receive(:static?).and_return(false)
-        allow(dbus).to receive(:start_mode).and_return(:manual)
       end
 
-      it "does not export the service as enable" do
-        expect(exported_services["enable"]).to_not include("dbus")
+      context "and is enabled" do
+        before do
+          allow(dbus).to receive(:start_mode).and_return(:on_boot)
+        end
+
+        it "exports the service as enabled" do
+          expect(exported_services["enable"]).to include("dbus")
+        end
+      end
+
+      context "and is disabled" do
+        before do
+          allow(dbus).to receive(:start_mode).and_return(:manual)
+        end
+
+        it "does not export the service as enable" do
+          expect(exported_services["enable"]).to_not include("dbus")
+        end
       end
     end
 
-    context "when service was disabled by user (has changes)" do
+    context "when service has changes (modified by user)" do
       before do
         allow(cups).to receive(:changed?).and_return(true)
-        allow(cups).to receive(:start_mode).and_return(:manual)
       end
 
-      it "exports the service as disabled" do
-        expect(exported_services["disable"]).to include("cups")
-      end
-    end
+      context "and was disabled" do
+        before do
+          allow(cups).to receive(:start_mode).and_return(:manual)
+        end
 
-    context "when service was not disabled by user" do
-      before do
-        allow(cups).to receive(:changed?).and_return(false)
-        allow(dbus).to receive(:start_mode).and_return(:manual)
+        it "exports the service as disabled" do
+          expect(exported_services["enable"]).to_not include("cups")
+          expect(exported_services["disable"]).to include("cups")
+        end
       end
 
-      it "does not exports the service" do
-        expect(exported_services["disable"]).to_not include("cups")
+      context "and was enabled" do
+        before do
+          allow(cups).to receive(:start_mode).and_return(:on_boot)
+        end
+
+        it "exports the service as enable" do
+          expect(exported_services["enable"]).to include("cups")
+          expect(exported_services["disable"]).to_not include("cups")
+        end
       end
 
-      context "but is proposed to be disabled" do
+      # FIXME this scenario should be fixed, see {#export} method
```

Comment:
yardoc does not know how to resolve the link to  `{#export}` here. Use the full name instead: `{Yast::ServicesManagerServiceClass#export}`

Reply:
Please, do not pay attention to my previous comment. I just realized that the comment it is in the test file :see_no_evil: